examples/nextjs/src/app/[[...slug]]/page.js

```diff
@@ -30,7 +31,9 @@ export default async function Home({ searchParams, params }) {
         path: params?.slug ? params.slug.join('/') : 'index',
         language_id: searchParams.language_id,
         personaId: searchParams['com.dotmarketing.persona.id'] || '',
-        mode: searchParams.mode
+        mode: searchParams.mode,
+        // Get the correct page to render
+        variantName: searchParams['variantName'] ? searchParams['variantName'] : 'DEFAULT'
```

Comment:
do we need to explicitly pass the `DEFAULT`?

Reply:
I think, no necessary, let me try to remove it.

Reply:
Fixed, more or less 🤣